language_model/ContentItem[] content_items